index.html

```diff
@@ -1,22 +1,30 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ja">
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" type="text/css" />
-    <title>TerraceTechフロントエンドエンジニア養成所（修正済みです！）</title>
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
   </head>
   <body>
     <div>
-      <ul id="js-list"></ul>
+      <ul id="js-list">
+      </ul>
     </div>
-    <script>
-      const ul = document.getElementById('js-list');
-      const li = document.createElement('li'); const a = document.createElement('a'); const img = document.createElement('img');
-      a.setAttribute("href","1.html"); img.setAttribute("src","bookmark.png"); img.setAttribute("alt","ブックマーク"); 
-      a.textContent = 'これです';
-      ul.appendChild(li);  li.appendChild(a); a.appendChild(img);
+    <script>    
+      var ul = document.getElementById('js-list');
+      for(var i=1; i<=2; i++) {
+      var li = document.createElement('li');
+      li.textContent = 'a'+i;
+      ul.appendChild(li);
+      var a = document.createElement('a');
+      a.href= 'a'+i+'.html'
+      li.prepend(a);
+      var img = document.createElement('img');
+      img.src = '/img/bookmark.png';
+      a.appendChild(img);
+}
     </script>
   </body>
-</html>
+</html>
```

Comment:
Please Look.
csakyo/homework#15 (review)

Reply:
> If you don't expect to prepend, there is a method called
> There is a method called insertAdjacentElement. (IE compatible).
> 
> Since insertAdjacentElement is a method (function), you can specify the position of the tag to be inserted as the first argument.
> 
> ●insertAdjacentElement
> https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentElement
> 
> Also, when reviewing
> You can add a link to the issue to make it easier for the reviewer to see.
> 
> https://discord.com/channels/726252977928667179/818786882711322624/820072855483580476
> https://discord.com/channels/726252977928667179/818786882711322624/820073712165847070



> If you don't expect to prepend, there is a method called
> There is a method called insertAdjacentElement. (IE compatible).
> 
> Since insertAdjacentElement is a method (function), you can specify the position of the tag to be inserted as the first argument.
> 
> ●insertAdjacentElement
> https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentElement
> 
> Also, when reviewing
> You can add a link to the issue to make it easier for the reviewer to see.
> 
> https://discord.com/channels/726252977928667179/818786882711322624/820072855483580476
> https://discord.com/channels/726252977928667179/818786882711322624/820073712165847070

Thank you for letting me know the existence of insertAdjacentElement. I will do my best to solve the problem!
Also, thank you for telling me how to publicize GitHub.
I didn't know it, so it was very helpful.

Reply:
sorry. I couldn't understand the comment.
If you don't mind, I would appreciate it if you could tell me your intention.

Reply:
![sample](https://user-images.githubusercontent.com/60378833/124935146-585dc080-e040-11eb-8afb-611c234350ae.png)
・
・
・
![sample_02](https://user-images.githubusercontent.com/60378833/124935417-9bb82f00-e040-11eb-980a-6dd4c89053ce.png)

Sorry！
this mark in last line can delete.
It doesn't have such a deep meaning, so please think of it as a memo.

Reply:
I understand! And I finished the setting. Thank you for explaining with the image.